docs/crow/scripting.md

```diff
@@ -59,13 +59,21 @@ cd ~
 git clone https://github.com/monome/druid.git
 ```
 
+Once it's downloaded we can install druid. We need to enter the druid folder and install it
+```
+cd ~/druid
+pip3 install .
+```
+
+Now druid should be ready to use, you might need to close and reopen the terminal to get access to it.
```

Comment:
pip will warn you here if the binary location is not on your path, but it might be good to draw attention to that warning message?

Reply:
You mean when `~/.local/bin` or `/usr/local/bin` isn't on `$PATH`?
The hint I added here was more for tab-completion, but that's also a valid point. On the other hand if pip already tells the user do we need to document it as well?